torch/ao/quantization/quantizer/quantizer.py

```diff
@@ -124,35 +121,6 @@ class DerivedQuantizationSpec(QuantizationSpecBase):
     quant_max: Optional[int] = None
     qscheme: Optional[torch.qscheme] = None
 
-
-# In the absence of better name, just winging it with QuantizationConfig
-@dataclass(eq=True, frozen=True)
```

Comment:
I would have said that we should move these to quantizer_utils for now instead of xnnpack_quantizer_utils